Source/PeleLMeX_DeriveFunc.cpp

```diff
@@ -1418,15 +1421,16 @@ pelelmex_derlambda(
   auto mu = dummies.array(0);
   auto rhotheta = dummies.array(NUM_SPECIES + 1);
   auto const* ltransparm = a_pelelm->trans_parms.device_parm();
+  auto const* leosparm = a_pelelm->eos_parms.device_parm();
   amrex::Real LeInv = a_pelelm->m_Lewis_inv;
   amrex::Real PrInv = a_pelelm->m_Prandtl_inv;
   amrex::ParallelFor(
-    bx,
-    [do_fixed_Le, do_fixed_Pr, do_soret, LeInv, PrInv, rhoY, T, rhoD, rhotheta,
-     lambda, mu, ltransparm] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+    bx, [do_fixed_Le, do_fixed_Pr, do_soret, LeInv, PrInv, rhoY, T, rhoD,
```

Comment:
In general, it is interesting that lmex does the capture explicitly. In C we let the compiler decide with `=`.

Reply:
I think PeleLMeX is inconsistent about which way it does it. Being explicit can be more clear/helpful, but it does add a bit of annoyance in the code. So I just followed what was there.